src/ColorVectorSpace.jl

```diff
@@ -314,6 +314,10 @@ end
 (.*){T<:Number}(A::AbstractArray{T}, b::TransparentGray) = mul(A, b)
 (.*){T<:Number}(b::TransparentGray, A::AbstractArray{T}) = mul(b, A)
 
+varm{C<:AbstractGray}(v::AbstractArray{C}, s::AbstractGray; corrected::Bool=true) =
+        varm(map(gray,v),gray(s); corrected=corrected)
+real{C<:AbstractGray}(x::Type{C}) = C
```

Comment:
Right, I agree. I probably didn't state it very clearly, but my sense is that `real` is the companion of `Real`, and so what it should return is either a value satisfying `isa(x, Real)` or a type satisfying `T <: Real`.